specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -26,6 +26,10 @@
           "type": "string",
           "description": "Type of linked service."
         },
+        "version": {
+          "type": "string",
+          "description": "Version of the linked service."
+        },
```

Comment:
@tsaiggo could you check if the version in HDInsightOnDemandLinkedService is still necessary?

Reply:
Yes. It's necessary. This HDInsightOnDemandLinkedService is owned by HDInsight team. They have their own business logic. 

Btw, any impact for the downstream SDK?

Reply:
It's causing duplicate version properties for the generated code from versionTypePropertiesVersion, and codegen now rename the original version in `HDInsightOnDemandLinkedService` as `versionTypePropertiesVersion`
```
- Interface HDInsightOnDemandLinkedService has a new required parameter versionTypePropertiesVersion
```

Reply:
Which SDK and will customer be blocked?